niworkflows/data/nipreps.json

```diff
@@ -149,13 +153,18 @@
       "name": "cohort",
       "pattern": "[_/\\\\]+cohort-0*(\\d+)",
       "dtype": "int"
+    },
+    {
+      "name": "volcohort",
+      "pattern": "[_/\\\\]+volcohort-0*(\\d+)",
+      "dtype": "int"
```

Comment:
never noticed this until now, but why are these 0 padded?

Reply:
I believe an earlier version of pybids had something like that for run, so it parsed `int("1")` instead of `int("01")`. I don't think it was necessary or that it is still there.